tests/unit/preprocessor/_multimodel/test_multimodel.py

```diff
@@ -514,13 +514,37 @@ def test_unify_time_coordinates():
 class PreprocessorFile:
     """Mockup to test output of multimodel."""
 
-    def __init__(self, cube=None):
+    def __init__(self, cube=None, attributes=None):
         if cube:
             self.cubes = [cube]
+        if attributes:
+            self.attributes = attributes
 
     def wasderivedfrom(self, product):
         pass
 
+    def group(self, keys: list) -> str:
```

Comment:
isn't this the same function as in the source code? You can just import that one

Reply:
I am having some issues trying to mock this. And I don't have much time to focus on that. Does it bother you too much if it stays like this?

Reply:
no not at all, leave it like this :+1: